src/Storages/Statistics/StatisticsCountMinSketch.cpp

```diff
@@ -0,0 +1,88 @@
+#include <DataTypes/DataTypeLowCardinality.h>
+#include <DataTypes/DataTypeNullable.h>
+#include <IO/ReadHelpers.h>
+#include <IO/WriteHelpers.h>
+#include <Storages/Statistics/StatisticsCountMinSketch.h>
+
+#if USE_DATASKETCHES
+
+namespace DB
+{
+
+namespace ErrorCodes
+{
+extern const int ILLEGAL_STATISTICS;
+}
+
+static constexpr auto num_hashes = 8uz;
```

Comment:
The reader wonders how you arrived at these constants?

Kindly add a comment. If you can't think of a good reason for these constants, please add something along the lines of "Constants chosen based on rolling dices, we don't know how they affect statistics size and estimation quality." (<-- the next person who stumbles upon this might want to re-consider the choice of constants).